### PR TITLE
#1743 [Feature]: Other language support 

### PR DESCRIPTION
**Pull request by PeterTakahashi**
#1743 [Feature]: Other language support 

I was referring to sushiswap-interface.
https://github.com/sushiswap/sushiswap-interface

### CODE REVIEW

File: js/packages/web/babel.config.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+          {
+            runtime: 'automatic',
+            importSource: !isServer ? '@welldone-software/why-did-you-render' : 'react',
+          },
+        ],
+      ]
+    : ['next/babel']
+ 
+   return { presets, plugins: ['macros'] }
+}
```
Comment (mkarots) [inline js/packages/web/babel.config.js:19]:
```suggestion
}

```

Comment (mkarots) [reply js/packages/web/babel.config.js:19]: not super important i guess but a missing newline here

File: js/packages/web/locale/en.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  "Congratulations": "Congratulations",
+  "Got it": "Got it",
+  "Mint": "Mint",
+  "Mint to": "Mint to",
+  "My Items": "My Items",
+  "New editions have been minted please view your NFTs in": "New editions have been minted please view your NFTs in",
+  "Number of editions to mint": "Number of editions to mint",
+  "The amazing world of {name}": "The amazing world of {name}",
+  "Total cost": "Total cost"
+}
```
Comment (mkarots) [inline js/packages/web/locale/en.json:13]:
```suggestion
}

```

Comment (mkarots) [reply js/packages/web/locale/en.json:13]: same here

File: js/packages/web/locale/ja.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+  "Congratulations": "おめでとうございます",
+  "Got it": "",
+  "Mint": "発行",
+  "Mint to": "発行先",
+  "My Items": "マイアイテム",
+  "New editions have been minted please view your NFTs in": "",
+  "Number of editions to mint": "発行数",
+  "The amazing world of {name}": "ようこそ{name}の世界へ",
+  "Total cost": "トータルコスト"
+}
```
Comment (mkarots) [inline js/packages/web/locale/ja.json:13]:
```suggestion
}

```

Comment (mkarots) [reply js/packages/web/locale/ja.json:13]: and here :D i just left the suggestions in case you want to go for the change

Comment (mkarots) [reply js/packages/web/locale/ja.json:13]: but i dont think there is really a hard technical reason for it, other than convention i.e check this out https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Comment by stegaBOB** [comment]
This PR is great! Super sorry for taking so long to get to this! Would you mind cleaning up the conflicts? After that feel free to ping me and I'll merge this in! Thanks again @PeterTakahashi

File: js/packages/web/locale/en.json (hunk 2 above)
Comment (PeterTakahashi) [reply js/packages/web/locale/en.json:13]: ja.json and en.json is a file generated automatically by lingui.
So I will see if I can add a newline.

**Comment by PeterTakahashi** [comment]
@stegaBOB 
I will resolve the conflicts.

File: js/packages/web/locale/en.json (hunk 2 above)
Comment (mkarots) [reply js/packages/web/locale/en.json:13]: Ah right I didn't know they re autogenerated, thanks

File: js/packages/web/babel.config.js (hunk 1 above)
Comment (PeterTakahashi) [reply js/packages/web/babel.config.js]: I added newline here.

File: js/packages/web/locale/ja.json (hunk 3 above)
Comment (PeterTakahashi) [reply js/packages/web/locale/ja.json:79]: I recreated the json by lingui and it added a newline at the end.

File: js/packages/web/locale/en.json (hunk 2 above)
Comment (PeterTakahashi) [reply js/packages/web/locale/en.json:79]: I recreated the json by lingui and it added a newline at the end.

**Comment by stegaBOB** [comment]
Looks like it isn't working correctly on my end. Getting `Plurals for locale ja aren't loaded. Use i18n.loadLocaleData method to load plurals for specific locale. Using other plural rule as a fallback.
(anonymous) @ breadcrumbs.ts:111` in the browser console when I'm running on localhost.

**Comment by stegaBOB** [comment]
@PeterTakahashi if you do figure out why it isnt working and can get out a fix, feel free to ping me and I'll review it! Thanks againl.

**Pull request closed without merging**